### DISCUSSION TITLE
NRT support

### DISCUSSION THREAD

**Discussion by Sa1Gur** [xunit/xunit#2414] [Question] [answered]
Do XUnit APIs have been updated with NRT annotations fully? Because if I do this  `Assert.NotNull(test);` - analyzer still shows me a warning? Compile team thinks it's on XUnit side.

https://github.com/dotnet/roslyn/issues/57067

Can you clear this up?

**Comment by bradwilson** [comment, marked as answer]
Yes, these are available in the `2.4.2-pre.12` on NuGet: https://www.nuget.org/packages/xunit.assert/2.4.2-pre.12

**Reply by Sa1Gur** [reply]
Thank you for clearing this up!
